src/source/PeerConnection/PeerConnection.c

```diff
@@ -921,6 +921,20 @@ STATUS peerConnectionGetCurrentLocalDescription(PRtcPeerConnection pRtcPeerConne
     return retStatus;
 }
 
+/// @brief parses string of form "$number $whatever" returns $number as uint32
+/// @return 0 if value is not parsable or null
+UINT32 parseExtId(PCHAR extmapValue)
+{
+    UINT32 extid = 0;
+    if (extmapValue == NULL && STRCHR(extmapValue, ' ') == NULL) {
```

Comment:
Nit: We can probably define ' ' as : #define SPACE_CHAR     (PCHAR)(' '). It is very easy to make a typing mistake and remove the space if this function were to ever be modified. But again, I am not super particular about this.

Reply:
i can see the merit in doing so, but cant force myself to add even more defines :) 
feel free to fix it